jenkins-pipelines/manager/centos-manager-backup-azure.jenkinsfile

```diff
@@ -12,6 +12,7 @@ managerPipeline(
     scylla_version: '',
     azure_image_db: '/subscriptions/6c268694-47ab-43ab-b306-3c5514bc4112/resourceGroups/scylla-images/providers/Microsoft.Compute/images/ScyllaDB-5.0.2-0.20220807.299122e78-1-build-28',
     // scylla 5.0.2
+    // TODO: Advance to the latest enterprise once new builds will be created
```

Comment:
open a task, to when 2023.1 will be release, to do this change here... this comment will get lost, and we will never remember to change it, when time comes

Reply:
why not run it with 5.2 ? until them ?

Reply:
that would make sense, but we want to move it to use latest enterprise... but true that, until 2023.1 is released, and we can move Azure jobs to use 5.2

Reply:
Done
https://github.com/scylladb/qa-tasks/issues/1187

Reply:
Also advanced it to 5.2.1